lib/index.js

```diff
@@ -523,12 +525,16 @@ export function _checkPresentation(presentation) {
  * @param {object} options - The options.
  * @param {object} options.credential - An object that could be a
  *   VerifiableCredential.
- * @param {string} [options.now] - A string representing date time in
- *   ISO 8601 format. Defaults to current date time.
+ * @param {string|Date} [options.now] - A string representing date time in
+ *   ISO 8601 format or an instance of Date. Defaults to current date time.
+ *
  * @throws {Error}
  * @private
  */
 export function _checkCredential({credential, now = new Date().toISOString()}) {
+  if(now instanceof Date) {
+    now = now.toISOString();
```

Comment:
I think `now` should default to being `Date` -- and we should parse it if it's a string. We should also parse `issuanceDate` and `expirationDate` and use the parsed values for the comparison to avoid any edge cases where the comparison does the wrong thing when in string format.

Reply:
Addressed here [dbf559d](https://github.com/digitalbazaar/vc-js/pull/119/commits/dbf559d8fc6bc1fcd430c59560df3b487d1a64a5)